core/shared/src/main/scala/org/typelevel/log4cats/LoggerFactory.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats
+
+import scala.annotation.implicitNotFound
+
+@implicitNotFound("""
+Implicit not found for LoggerFactory[${F}].
+Information can be found here: https://log4cats.github.io/logging-capability.html
```

Comment:
Here is the dead link. We can pick up the fix from the main.
```suggestion
Learn about LoggerFactory at https://typelevel.org/log4cats/#logging-using-capabilities
```